### ISSUE TITLE
Overgrown Square Lights - Personal Opinion

### ISSUE THREAD

**Issue by Konclan** [BEEmod/BEE2-items#1293]
The square lights are very ugly and also don't fit the overgrown theme, why would they use florescent lights? I strongly feel these should be like the clean style square lights that have rust and dirt on them.

**Comment by LautaroL20** [comment]
Actually the "Instance Swapped" was a "Swapped Palette Icon" wich is now fixed. 
As for the Overgrown Square Lights, i don´t think they re ugly, they looks fine and ill planning on using these for the 80´s variant. They´re a custom model maded by TeamSpen210 BTW.

**Comment by Konclan** [comment]
Yes these could pass for old aperture but overgrown isn't old aperture, lights like these don't belong.

**Comment by LautaroL20** [comment]
Well, Over Square Lights using custom model was the TeamSpen210´s Idea. Also the Editor Model was swapped too.

**Comment by LautaroL20** [comment]
I have a simple idea for an Overgrown Square Lights version, and its using "Broken Glass" on them, but we will need to cut the model file so it can fit in the square cavity without crossing the limits.

**Comment by TeamSpen210** [comment]
The idea is that the front glass is worn/no longer there, so the glass is see-through and you can see the inside of the lights.

**Comment by Alfe5** [comment]
i agree they should be reworked. They should look more like the clean style with a bit of dirt and damage

**Comment by Luke18033** [comment]
Maybe make the glass cracked.

**Comment by TeamSpen210** [comment]
They do need some reworking, to make them look more similar.

**Comment by Luke18033** [comment]
Maybe use a custom skin/model to make it cracked.
